Troubleshooting.md

```diff
@@ -0,0 +1,210 @@
+We've listed all the known issues with the CORTX software and the steps to resolve them below. Please refer to our [support documentation](SUPPORT.md) to report new issues or want us to document steps to resolve an issue. 
+
+# Known Issues
+
+<details>
+<summary>Firmware bundle upload and update functionality not working</summary>
```

Comment:
I have no idea what this means.  Do you?

Reply:
Fixed it.